### DISCUSSION TITLE
Can't get the student pack (technical difficulty?)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#53456] [GitHub Education]
Hi, when I enter all the information and click on submit, it loads indefinitely and the console produces following error: 'Failed to load resource: the server responded with a status of 404 ()'.
Is anybody else experiencing this? I am on a Mac if it helps...

**Comment by atakhan** [comment]
i have a same problem.. (

**Comment by ghost** [comment]
same problem here, macos safari

**Comment by samus-aran** [comment]
Hi folks 👋 Are you still experiencing this? If so raise a EDU Support ticket by selecting "Other" and provide as much information/screenshots/browser logs as possible in your support ticket. 👍 Jump back here and update this discussion so we can associate it with your ticket !

**Reply by ghost** [reply]
Hi, I filed a Support Ticket but haven't gotten a response after replying to an automated answer. Since then it has been 3 days

**Comment by ghost** [comment]
Still hasn't been resolved and no further notices from support after automated answer...
